taipy/gui/_renderers/utils.py

```diff
@@ -36,9 +37,13 @@ def _get_columns_dict_from_list(
             col_dict[col] = {"index": idx}
             idx += 1
         elif col:
-            _warn(
```

Comment:
I think it would be better if we calculate the list of columns depending on the type of value here
something like 
`cols = str(list(value.columns) if isinstance(value, dataframe) else list(value.keys()) if isinstance(value, (dict, _MapDict)) else value if isinstance(value, (list, tuple)) ... else value)`

Reply:
> I think it would be better if we calculate the list of columns depending on the type of value here
> 
> something like 
> 
> `cols = str(list(value.columns) if isinstance(value, dataframe) else list(value.keys()) if isinstance(value, (dict, _MapDict)) else value if isinstance(value, (list, tuple)) ... else value)`

Oh I can do that I thought we were just using data frames. What all data types does the function support?

Reply:
dataframe, dict, list ... I suppose this would be enough
`cols = str(list(value.columns) if isinstance(value, dataframe) else list(value.keys()) if isinstance(value, (dict, _MapDict)) else value if isinstance(value, (list, tuple)) else value)
`

Reply:
Added the required changes

Reply:
I can't see them ?